src/table/__tests__/a11y.test.tsx

```diff
@@ -180,5 +180,28 @@ describe('labels', () => {
         `Displaying items from ${firstIndex} to ${lastIndex} of ${totalItemsCount} items`
       );
     });
+
+    test('The table items live region must not include nested items', () => {
```

Comment:
This change is unrelated, looks identical to the test before - likely caused by the merge resolution.